addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderServerBridgeHandler.java

```diff
@@ -618,276 +606,692 @@ private void onDisconnected() {
 
             if (thingHandler != null) {
                 thingHandler.onBridgeDisconnected(this);
-                logger.debug("{}: onDisconnected(): Bridge - {}, Thing - {}, Thing Handler - {}", getLogIdentifier(),
-                        thing.getBridgeUID(), thing.getUID(), thingHandler);
+                logger.debug("{}: context='onDisconnected': Bridge - {}, Thing - {}, Thing Handler - {}",
+                        getLogIdentifier(), thing.getBridgeUID(), thing.getUID(), thingHandler);
             }
         }
     }
 
-    @Override
-    public void updateAvaliabilityStatus(IZoneMinderConnectionInfo connection) {
+    private int initRetriesCount = 0;
```

Comment:
Please make sure that all the fields are one top of the class, or split up in different classes which have a higher cohesion